### PR TITLE
Table are responsive (resize when screen resize)

### PR DESCRIPTION
**Pull request by dacolgit**
fix Table border disappears when hovered over a short text
add reload value for node variable in drawer
UI fix

### CODE REVIEW

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Gateways.razor [hunk 1]
```diff
@@ -26,20 +26,32 @@
 <table class="table @_tableView">
     <thead>
         <tr>
-            <th>Id</th>
-            <th>Site Id</th>
-            <th>Connection Status</th>
+            <th class="x-large">Id</th>
+            <th class="large">Site Id</th>
```
Comment (MaidaB) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Gateways.razor:30]: Should here be width-large?

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor [hunk 2]
```diff
@@ -113,33 +113,49 @@
...
         {
         <tr>
-            <td>@node.NodeName</td>
-            <td>@node.Id</td>
+            <td class="hover-text width-medium">
+                <div>
+                    @node.NodeName
+                </div>
+            </td>
+            <td class="hover-text width-large1">
```
Comment (cristipogacean) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:121]: >large1 [](start = 40, length = 6)

is this 1 in "...large1" correct? or should it be just "...large"

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor [hunk 3]
```diff
@@ -113,33 +113,49 @@
...
+                    </div>
+                </td>
             }
             @if (node.NodeClass == NodeClass.Variable && (((node.AccessLevel & NodeAccessLevel.CurrentRead) == NodeAccessLevel.CurrentRead) ||
                ((node.AccessLevel & NodeAccessLevel.CurrentWrite) == NodeAccessLevel.CurrentWrite)))
             {
-                <td class="value-wide value-scroll">
-                    <div>
+                <td class="hover-text width-large1">
+                    <div class="value-scroll">
```
Comment (MaidaB) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:158]: Here we get extremely long lines 

image

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor (hunk 2 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:121]: it's correct, but I rename it

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Gateways.razor (hunk 1 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Gateways.razor:30]: yes thanks

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor (hunk 3 above)
Comment (dacolgit) [reply samples/src/Microsoft.Azure.IIoT.App/src/Pages/Browser.razor:158]: fixed

File: samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor [hunk 4]
```diff
@@ -62,21 +62,23 @@
...
+                    <td class="hover-text">@discoverer.DiscovererModel.Id</td>
                 }
                 @{connectStatus = discoverer.DiscovererModel.Connected == true ? "Connected" : "Disconnected";}
-                <td class="td-short">
-                    @connectStatus
+                <td class="hover-text width-medium">
+                    <div>@connectStatus</div>
                 </td>
-                <td class="td-short">
+                <td class="medium">
```
Comment (MaidaB) [inline samples/src/Microsoft.Azure.IIoT.App/src/Pages/Discoverers.razor:81]: width-medium?

**Pull request merged**